_articles/legal.md

```diff
@@ -156,3 +158,11 @@ Longer term, your legal team can do more to help the company get more from its i
 
 * **Patents:** Your company may wish to join the [Open Invention Network](https://www.openinventionnetwork.com/), a shared defensive patent pool to protect members' use of major open source projects, or explore other [alternative patent licensing](https://www.eff.org/document/hacking-patent-system-2016).
 * **Governance:** Especially if and when it makes sense to move a project to a [legal entity outside of the company](../leadership-and-governance/#do-i-need-a-legal-entity-to-support-my-project).
+
+## What if someone uses my project without following the license terms?
+
+<!-- 
+  TODO 
+   - Talk about **registering a copyright** to claim "statutory damages" since otherwise claims would only be able to cover "actual damages", which are much harder to argue for from a free, open-source project.
+   - Talk about litigation costs. Consider mentioning foundations which help open source developers (FSF, EFF, sfconservancy)[https://github.com/community/maintainers/discussions/341#discussioncomment-6491105]
+-->
```

Comment:
The linked Maintainer Community discussion highlights what I consider this article's biggest gap -- how to handle infringement.

The obvious answer is "get a lawyer", but I think it would be helpful to provide links to OSS-specific foundations/communities that help with litigation.